client/js/lounge.js

```diff
@@ -750,6 +763,18 @@ $(function() {
 		case "user":
 			$(".channel.active .users .user[data-name=" + $(this).data("data") + "]").click();
 			break;
+		case "user-ban":
+			socket.emit("input", {
+				target: chat.data("id"),
+				text: "/mode +b " + $(this).data("data")
+			});
+		// eslint-disable-next-line no-fallthrough
```

Comment:
I was assuming only these two options were wanted. Having all three would be ideal, yes.